### DISCUSSION TITLE
Is there a way to force the card to download the most up-to-date map?

### DISCUSSION THREAD

**Discussion by ezfrag2021** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#404] [General]
Love the card, by the way. For someone like me for whom this would be impossible to do myself it's incredible that great people are willing to share their hard work. Thank you!

I have two vacuum cleaners all set-up in HA. However, neither of them are giving me the option of choosing "room cleaning". I only get the option of zone cleanup or pin and go. I assume that this was because I hadn't properly labelled the rooms in the Xiaomi App so they weren't pulling through. I have edited the maps in the Xiaomi App and now need to force a refresh in HA. It looks like the Map Extractor does this automatically but the maps look identical and I still don't have any room options on the Lovelace card. Any suggestions? Cheers!

<img width="1072" alt="Screenshot 2022-05-27 at 10 02 35" src="https://user-images.githubusercontent.com/93487625/170680488-198dc8d9-7d92-4558-969c-b2cba6208554.png">

**Comment by PiotrMachowski** [comment]
Card doesn't create rooms config for you, you have to do it manually: https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/discussions/317

**Reply by ezfrag2021** [reply]
Sorry I must have missed that. Thanks for guiding me to it :)
